src/pip/_internal/locations/_sysconfig.py

```diff
@@ -214,5 +214,9 @@ def get_platlib() -> str:
 
 
 def get_prefixed_libs(prefix: str) -> typing.Tuple[str, str]:
-    paths = sysconfig.get_paths(vars={"base": prefix, "platbase": prefix})
+    vars = {"base": prefix, "platbase": prefix}
+    if "venv" in sysconfig.get_scheme_names():
+        paths = sysconfig.get_paths(vars=vars, scheme="venv")
+    else:
+        paths = sysconfig.get_paths(vars=vars)
     return (paths["purelib"], paths["platlib"])
```

Comment:
Since this function is only used for creating isolated environments, let’s rename it to better describe what it does. This avoids needing to think about why this always sets `venv` even when pip is not running in a virtual environment.

Reply:
I thought `prefixed` in the name carried this message. Do you have a better name in mind?

Reply:
Maybe just `get_isolated_environment_lib_paths`?

Reply:
Sounds descriptive enough. Should I change the argument name to `environment` then?

Reply:
`prefix` is probably fine (it is the environment’s path prefix after all)

Reply:
I just noticed that the function name is meant to reflect the name of a similar function in `distutils`. It is also called from a function with the same name in another module, see https://github.com/pypa/pip/blob/9aa422da16e11b8e56d3597f34551f983ba9fbfd/src/pip/_internal/locations/__init__.py#L485 Should we still change just this one?

Reply:
No, change them all.